deploy/jN/includes/CSSLint.js

```diff
@@ -1,5 +1,67 @@
 (function(){
 
+function createDialog(cfg){
+	var header = "";
+	for(var i=0; i<cfg.header.length; i++)
+		header +="<th>"+cfg.header[i]+"</th>";
```

Comment:
and this one too. Should definitely try to improve this. But I don't remember currently why don't I use DOM API here and if it is possible at allmor not. In fact I have modularized that very long ago.